copy-project.rst

```diff
@@ -0,0 +1,27 @@
+.. _copy-project:
+
+How to duplicate a project
+==========================
+
+To help you re-use work and speed up your workflow, Divio offers multiple options for
+project duplication - Copy, Fork and Mirror.
+
+Select your duplication option
+------------------------------
+
+To duplicate a project, open its actions menu and select Duplicate.
+
+You need to give the duplicate a name, and decide whether to duplicate the project to a different organisation.
+
+From the *Copy type* select the appropriate duplication action, Copy, Fork or Mirror. Refer to :ref:`duplicating projects <project-duplication>`
```

Comment:
Please set your editor to re-wrap lines at 100 or 120 columns, and reformat paragraphs to fit into that limit.